docs/writing_a_parser_plugin/parser_shell.md

```diff
@@ -0,0 +1,128 @@
+# Creating the shell for your parser
```

Comment:
Be careful with the choice of words: **shell** is often used to refer to a terminal session (actually 1 layer deeper). I would use **Setting up the framework for your parser**

Reply:
changed.